listenbrainz/webserver/views/api_compat_deprecated.py

```diff
@@ -67,42 +68,63 @@ def handshake():
         'OK',
         session.sid,
         '{}/np_1.2'.format(config.LASTFM_PROXY_URL),
-        '{}/protocol_1.2'.format(config.LASTFM_PROXY_URL)
+        '{}/protocol_1.2\n'.format(config.LASTFM_PROXY_URL)
```

Comment:
I read VLC's audioscrobbler plugin source code ([here](https://github.com/videolan/vlc/blob/master/modules/misc/audioscrobbler.c#L626)) and it seems that they require `\n` at the end of each line, I think this might be the reason why vlc wasn't able to send submissions to production even though it worked locally for me. (not sure though)